app/internal/page/identity_options.go

```diff
@@ -97,22 +97,23 @@ type IdentityOptions struct {
 	Selected []IdentityOption
 	First    IdentityOption
 	Second   IdentityOption
+	Paths    AppPaths
```

Comment:
I'm a little bit iffy about this approach. Although it's going to be rare for our URLs to change there may be a need to do this in the future and then as we persist identity options on the Lpa struct we could have out-of-date paths. We could potentially rework the flow to query whatever the path should be as its needed but wanted to get some feedback first (and would also be a future ticket)

Reply:
Much better - actioned ✅